# -*- coding: utf-8 -*-

# rolilahla
#Pycharm

from PyQt5 import QtCore, QtGui, QtWidgets
from dosyaolustur import Ui_DosyaOlustur

class Ui_MainWindow(object):
    def dosya_olustur_gui_hazirla(self):
        pass

    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(929, 490)
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("../../../Downloads/faenza-icon-theme-master/Faenza/places/48/distributor-logo-ubuntu.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        MainWindow.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.gridLayout = QtWidgets.QGridLayout(self.centralwidget)
        self.gridLayout.setObjectName("gridLayout")
        self.tabWidget = QtWidgets.QTabWidget(self.centralwidget)
        self.tabWidget.setObjectName("tabWidget")
        self.tab = QtWidgets.QWidget()
        self.tab.setObjectName("tab")
        self.label = QtWidgets.QLabel(self.tab)
        self.label.setGeometry(QtCore.QRect(10, 10, 71, 20))
        self.label.setObjectName("label")
        self.comboBox = QtWidgets.QComboBox(self.tab)
        self.comboBox.setGeometry(QtCore.QRect(70, 10, 101, 20))
        self.comboBox.setObjectName("comboBox")
        self.comboBox.addItem("")
        self.groupBox = QtWidgets.QGroupBox(self.tab)
        self.groupBox.setGeometry(QtCore.QRect(10, 230, 191, 141))
        self.groupBox.setObjectName("groupBox")
        self.gridLayout_2 = QtWidgets.QGridLayout(self.groupBox)
        self.gridLayout_2.setObjectName("gridLayout_2")
        self.tableWidget = QtWidgets.QTableWidget(self.groupBox)
        font = QtGui.QFont()
        font.setPointSize(8)
        self.tableWidget.setFont(font)
        self.tableWidget.setObjectName("tableWidget")
        self.tableWidget.setColumnCount(2)
        self.tableWidget.setRowCount(4)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setVerticalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setVerticalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setVerticalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setVerticalHeaderItem(3, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setHorizontalHeaderItem(1, item)
        self.tableWidget.horizontalHeader().setDefaultSectionSize(77)
        self.tableWidget.horizontalHeader().setMinimumSectionSize(21)
        self.tableWidget.verticalHeader().setDefaultSectionSize(21)
        self.tableWidget.verticalHeader().setMinimumSectionSize(19)
        self.gridLayout_2.addWidget(self.tableWidget, 0, 0, 1, 1)
        self.groupBox_2 = QtWidgets.QGroupBox(self.tab)
        self.groupBox_2.setGeometry(QtCore.QRect(10, 40, 191, 121))
        self.groupBox_2.setCursor(QtGui.QCursor(QtCore.Qt.ArrowCursor))
        self.groupBox_2.setFlat(False)
        self.groupBox_2.setCheckable(False)
        self.groupBox_2.setObjectName("groupBox_2")
        self.groupBox_3 = QtWidgets.QGroupBox(self.tab)
        self.groupBox_3.setGeometry(QtCore.QRect(10, 170, 191, 51))
        self.groupBox_3.setObjectName("groupBox_3")
        self.label_7 = QtWidgets.QLabel(self.groupBox_3)
        self.label_7.setGeometry(QtCore.QRect(50, 20, 111, 20))
        font = QtGui.QFont()
        font.setPointSize(9)
        font.setBold(True)
        font.setWeight(75)
        self.label_7.setFont(font)
        self.label_7.setObjectName("label_7")
        self.groupBox_4 = QtWidgets.QGroupBox(self.tab)
        self.groupBox_4.setGeometry(QtCore.QRect(230, 0, 600, 100))
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.groupBox_4.sizePolicy().hasHeightForWidth())
        self.groupBox_4.setSizePolicy(sizePolicy)
        self.groupBox_4.setObjectName("groupBox_4")
        self.gridLayout_3 = QtWidgets.QGridLayout(self.groupBox_4)
        self.gridLayout_3.setObjectName("gridLayout_3")
        self.textEdit = QtWidgets.QTextEdit(self.groupBox_4)
        self.textEdit.setObjectName("textEdit")
        self.gridLayout_3.addWidget(self.textEdit, 0, 0, 1, 1)
        self.groupBox_5 = QtWidgets.QGroupBox(self.tab)
        self.groupBox_5.setGeometry(QtCore.QRect(230, 105, 600, 100))
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.groupBox_5.sizePolicy().hasHeightForWidth())
        self.groupBox_5.setSizePolicy(sizePolicy)
        self.groupBox_5.setObjectName("groupBox_5")
        self.gridLayout_4 = QtWidgets.QGridLayout(self.groupBox_5)
        self.gridLayout_4.setObjectName("gridLayout_4")
        self.textEdit_2 = QtWidgets.QTextEdit(self.groupBox_5)
        self.textEdit_2.setObjectName("textEdit_2")
        self.gridLayout_4.addWidget(self.textEdit_2, 0, 0, 1, 1)
        self.groupBox_6 = QtWidgets.QGroupBox(self.tab)
        self.groupBox_6.setGeometry(QtCore.QRect(230, 210, 600, 100))
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.groupBox_6.sizePolicy().hasHeightForWidth())
        self.groupBox_6.setSizePolicy(sizePolicy)
        self.groupBox_6.setObjectName("groupBox_6")
        self.gridLayout_5 = QtWidgets.QGridLayout(self.groupBox_6)
        self.gridLayout_5.setObjectName("gridLayout_5")
        self.textEdit_3 = QtWidgets.QTextEdit(self.groupBox_6)
        self.textEdit_3.setObjectName("textEdit_3")
        self.gridLayout_5.addWidget(self.textEdit_3, 0, 0, 1, 1)
        self.pushButton = QtWidgets.QPushButton(self.tab)
        self.pushButton.setGeometry(QtCore.QRect(840, 40, 51, 41))
        self.pushButton.setObjectName("pushButton")
        self.pushButton_2 = QtWidgets.QPushButton(self.tab)
        self.pushButton_2.setGeometry(QtCore.QRect(840, 140, 51, 41))
        self.pushButton_2.setObjectName("pushButton_2")
        self.pushButton_3 = QtWidgets.QPushButton(self.tab)
        self.pushButton_3.setGeometry(QtCore.QRect(840, 240, 51, 41))
        self.pushButton_3.setObjectName("pushButton_3")
        self.label_2 = QtWidgets.QLabel(self.tab)
        self.label_2.setGeometry(QtCore.QRect(20, 60, 81, 20))
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(self.tab)
        self.label_3.setGeometry(QtCore.QRect(20, 80, 71, 20))
        self.label_3.setObjectName("label_3")
        self.label_4 = QtWidgets.QLabel(self.tab)
        self.label_4.setGeometry(QtCore.QRect(20, 100, 71, 20))
        self.label_4.setObjectName("label_4")
        self.label_5 = QtWidgets.QLabel(self.tab)
        self.label_5.setGeometry(QtCore.QRect(20, 120, 71, 20))
        self.label_5.setObjectName("label_5")
        self.label_6 = QtWidgets.QLabel(self.tab)
        self.label_6.setGeometry(QtCore.QRect(20, 140, 71, 20))
        self.label_6.setObjectName("label_6")
        self.tabWidget.addTab(self.tab, "")
        self.gridLayout.addWidget(self.tabWidget, 0, 1, 1, 1)
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 929, 21))
        self.menubar.setObjectName("menubar")
        self.menuDosya_Ekle = QtWidgets.QMenu(self.menubar)
        self.menuDosya_Ekle.setObjectName("menuDosya_Ekle")
        self.menuUzla_mac_Ekle = QtWidgets.QMenu(self.menubar)
        self.menuUzla_mac_Ekle.setObjectName("menuUzla_mac_Ekle")
        self.menu_ablonlar = QtWidgets.QMenu(self.menubar)
        self.menu_ablonlar.setObjectName("menu_ablonlar")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)
        self.actionDosya_Ekle = QtWidgets.QAction(MainWindow)
        self.actionDosya_Ekle.setObjectName("actionDosya_Ekle")
        self.actionUzla_mac_Bilgisi_ekle = QtWidgets.QAction(MainWindow)
        self.actionUzla_mac_Bilgisi_ekle.setObjectName("actionUzla_mac_Bilgisi_ekle")
        self.action_ablonlar_d_zenle = QtWidgets.QAction(MainWindow)
        self.action_ablonlar_d_zenle.setObjectName("action_ablonlar_d_zenle")
        self.menuDosya_Ekle.addAction(self.actionDosya_Ekle)
        self.menuUzla_mac_Ekle.addAction(self.actionUzla_mac_Bilgisi_ekle)
        self.menu_ablonlar.addAction(self.action_ablonlar_d_zenle)
        self.menubar.addAction(self.menuDosya_Ekle.menuAction())
        self.menubar.addAction(self.menuUzla_mac_Ekle.menuAction())
        self.menubar.addAction(self.menu_ablonlar.menuAction())

        self.retranslateUi(MainWindow)
        self.tabWidget.setCurrentIndex(0)
        self.comboBox.activated['QString'].connect(self.tabWidget.close)
        self.pushButton.clicked.connect(MainWindow.close)
        self.pushButton_2.clicked.connect(self.statusbar.close)
        self.pushButton_3.clicked.connect(self.statusbar.close)

        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "Uzlaş uzlaşabilirsen"))
        self.label.setText(_translate("MainWindow", "Dosya No"))
        self.comboBox.setItemText(0, _translate("MainWindow", "Dosya Seç"))
        self.groupBox.setTitle(_translate("MainWindow", "Taraflar"))
        item = self.tableWidget.verticalHeaderItem(0)
        item.setText(_translate("MainWindow", "1"))
        item = self.tableWidget.verticalHeaderItem(1)
        item.setText(_translate("MainWindow", "2"))
        item = self.tableWidget.verticalHeaderItem(2)
        item.setText(_translate("MainWindow", "3"))
        item = self.tableWidget.verticalHeaderItem(3)
        item.setText(_translate("MainWindow", "4"))
        item = self.tableWidget.horizontalHeaderItem(0)
        item.setText(_translate("MainWindow", "Ad Soyad"))
        item = self.tableWidget.horizontalHeaderItem(1)
        item.setText(_translate("MainWindow", "Sıfat"))
        self.groupBox_2.setTitle(_translate("MainWindow", "Dosya Bilgileri"))
        self.groupBox_3.setTitle(_translate("MainWindow", "Dosya Durumu"))
        self.label_7.setText(_translate("MainWindow", "Görüşme halinde"))
        self.groupBox_4.setTitle(_translate("MainWindow", "Uzlaşma Görüşmeleri"))
        self.groupBox_5.setTitle(_translate("MainWindow", "Edim"))
        self.groupBox_6.setTitle(_translate("MainWindow", "Uzlaşma Başarısız Olmuşsa Sebebleri"))
        self.pushButton.setText(_translate("MainWindow", "Güncelle"))
        self.pushButton_2.setText(_translate("MainWindow", "Güncelle"))
        self.pushButton_3.setText(_translate("MainWindow", "Güncelle"))
        self.label_2.setText(_translate("MainWindow", "Soruşturma No"))
        self.label_3.setText(_translate("MainWindow", "M.Esas No"))
        self.label_4.setText(_translate("MainWindow", "Suç"))
        self.label_5.setText(_translate("MainWindow", "Suç Tarihi"))
        self.label_6.setText(_translate("MainWindow", "Dosya No"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab), _translate("MainWindow", "Dosya Bilgileri"))
        self.menuDosya_Ekle.setTitle(_translate("MainWindow", "Dosya Ekle"))
        self.menuUzla_mac_Ekle.setTitle(_translate("MainWindow", "Uzlaşmacı "))
        self.menu_ablonlar.setTitle(_translate("MainWindow", "Şablonlar"))
        self.actionDosya_Ekle.setText(_translate("MainWindow", "Dosya Ekle"))
        self.actionUzla_mac_Bilgisi_ekle.setText(_translate("MainWindow", "Uzlaşmacı Ekle"))
        self.action_ablonlar_d_zenle.setText(_translate("MainWindow", "Şablonları düzenle"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    MainWindow = QtWidgets.QMainWindow()
    ui = Ui_MainWindow()
    ui.setupUi(MainWindow)
    MainWindow.show()
    sys.exit(app.exec_())

